### DISCUSSION TITLE
I don’t want to compile the source code, how do I need to install it, I’m centos

### DISCUSSION THREAD

**Discussion by RunlaiDeng** [filecoin-project/lotus#5940] [General Q&A] [answered]
I don’t want to compile the source code, how do I need to install it, I’m centos

**Comment by RunlaiDeng** [comment]
![image](https://user-images.githubusercontent.com/30950645/113379195-68179a00-93ab-11eb-99bb-b957a3ed29af.png)
The cost of compiling source code is very high

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
as for now you have to compile the lotus software family by yourself. there are no binary releases - and i doubt there ever will be. compiling the code on your own gives you certain securities and mitigates certain supply chain attack scenarios - short: it is safer.

btw. the cost of compiling are quite low compared to running the node or the miner software.
